schema/data/project-page-data.jsonld

```diff
@@ -0,0 +1,242 @@
+{
+  "id": "http://dev.app.regen.com/projects/38",
+  "environmentalAuthority": {
+    "name": "Corantioquia",
+    "url": "http://corantioquia.gov.co/"
+  },
+  "marketType": "rfs:ComplianceMarket",
+  "bioregion": [
+    "Amazon Basin"
+  ],
+  "biomeType": [
+    "TropicalForest"
+  ],
+  "watershed": [
+    "Amazon River"
+  ],
+  "subWatershed": [
+    "Upper Amazon"
+  ],
+  "ecosystemType": [
+    "rfs:CloudForest"
+  ],
+  "offchainCreditsInfo": {
+    "creditsRegistered": {
+      "numericValue": 1000.0,
+      "unit": "unit:HA"
+    },
+    "creditsAvailable": {
+      "numericValue": 800.0,
+      "unit": "unit:HA"
+    },
+    "creditsRetired": {
+      "numericValue": 200.0,
+      "unit": "unit:HA"
+    }
+  },
+  "projectDuration": "P1Y",
+  "projectDurationMinimum": "P6M",
+  "projectDurationMaximum": "P2Y",
+  "managementAreas": [
+    {
+      "activity": [
+        "rfs:Conservation"
+      ],
+      "extent": {
+        "numericValue": 120.0,
+        "unit": "unit:HA"
+      }
+    },
+    {
+      "activity": [
+        "rfs:EcosystemRestoration"
+      ],
+      "extent": {
+        "numericValue": 220.0,
+        "unit": "unit:HA"
+      }
+    }
+  ],
+  "ecologicalConnectivityIndex": {
+    "numericValue": 75.5,
+    "unit": "unit:UNITLESS"
+  },
+  "socialCulturalIndex": {
+    "numericValue": 85.0,
+    "unit": "unit:UNITLESS"
+  },
+  "administrativeArea": {
+    "name": "Antioquia"
+  },
+  "projectOperator": {
+    "name": "El Globo",
+    "showOnProjectPage": true,
+    "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ",
+    "image": "https://via.placeholder.com/150"
+  },
+  "projectMonitor": {
+    "name": "EAFIT",
+    "showOnProjectPage": true,
```

Comment:
```suggestion
```

Since all of these are defined in the project page metadata, I don't know if we need this explicitly set. Maybe @blushi has thoughts?